### DISCUSSION TITLE
Files GUI & Samba

### DISCUSSION THREAD

**Discussion by FritzLott** [Security-Onion-Solutions/securityonion#5833] [Unsupported Versions] [answered]
I would like to use Wireshark to capture traffic and then move the capture files to my laptop for analysis.  It's awkward (but doable) to move the capture files up to Google Drive and then download them to my laptop.  The Files GUI application allows a LAN connection to a Samba NAS (network attached storage) but the connect button is grayed out.  (The button works quite nicely on my CentOS 7 laptop.)

I have tried installing Samba and Samba-client but without impacting that button. The button stays stubbornly gray.  I can see my Samba NAS with smbclient -L  ip-address.  So the firewall is not stopping me.

Does anyone have a suggestion on how to deal with the connect button in Files GUI?

**Comment by FritzLott** [comment, marked as answer]
"yum install nautilus" shows that Gnome Files is version [nautilus-3.26.3.1-7.el7.x86_64.rpm](https://centos.pkgs.org/7/centos-x86_64/nautilus-3.26.3.1-7.el7.x86_64.rpm.html)
"yum install gvfs-smb" removes the greyed button to connect with the server "smb://ip_address/Share"

**Comment by FritzLott** [comment]
I forgot the link to rectify the greyed out button:
https://forums.centos.org/viewtopic.php?t=69829

**Comment by dougburks** [comment]
This seems like a strange deployment. Have you considered just doing a normal distributed deployment?
https://docs.securityonion.net/en/2.3/architecture.html#distributed
